releases/sui/2025-01-ITS-v1.1.0.md

```diff
@@ -80,55 +79,43 @@ kubectl exec genesis-0 -- axelard tx wasm execute "$ITS_HUB_ADDR" \
 --gas-adjustment 1.2
 ```
 
-#### Stagenet/Testnet:
+#### Stagenet/Testnet/Mainnet:
```

Comment:
Avoid using different approaches for devnet vs other networks. Execute via a proposal for devnet as well. Edwin should be adding a script to execute directly instead that we can use in the future

Reply:
Switch the cmd to use governance for now for devnet as well, until Edwin improves the script